app/emails/templates/team_lead_added.md

```diff
@@ -6,7 +6,7 @@ You've been added as a team lead of {{ team_name }}. This means you can:
 * add and remove other team leads
 * set permissions for each member
   
-[Sign in]({{ frontend_url }}/login) to the Digital Marketplace to view your team's opportunities on your dashboard.  
+[Sign in]({{ frontend_url }}/2/login) to the Digital Marketplace to view your team's opportunities on your dashboard.  
```

Comment:
What about the `team_member_added` template?

Reply:
add the new login to` team_member_added` template